myhdl/_Signal.py

```diff
@@ -26,6 +26,8 @@
 negedge -- callable to model a falling edge on a signal in a yield statement
 
 """
+from __future__ import absolute_import
+from __future__ import print_function
```

Comment:
To the PR comment, shouldn't we standardize `absolute_import`, `print_function`, and `division` in all files as prep for py3k?